cli.js

```diff
@@ -62,12 +74,10 @@ const argv = require('yargs')
           .option('pattern', { describe: 'Match pattern will filter baseline', default: null, type: 'string' })
       },
       async (argv) => {
-        const log = logger.create(argv.verbose);
-        const f = filter.create(log);
-        await f.configure();
-        log.con(f.filter);
+        const {l, f, t, a} = await scope(argv.verbose, argv.presetsDest);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "a" [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://159.65.230.204:9000/coding_rules#rule_key=javascript%3AS1854)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "t" [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://159.65.230.204:9000/coding_rules#rule_key=javascript%3AS1854)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of the unused 't' variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://159.65.230.204:9000/coding_rules#rule_key=javascript%3AUnusedVariable)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of the unused 'a' variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://159.65.230.204:9000/coding_rules#rule_key=javascript%3AUnusedVariable)